packages/docusaurus-utils/index.js

```diff
@@ -47,6 +49,11 @@ function fileToComponentName(file) {
   return ext ? ext.toUpperCase() + str : str;
 }
 
+function generateChunkName(str, prefix) {
+  const name = str === `/` ? `index` : kebabHash(str);
```

Comment:
Why are we using backticks for strings now? We could change all to that but it's not very common practice I'm afraid. Only app I've seen do it is Gatsby which also likes to omit semi-colons and I'm not a fan of that. :/

Reply:
My bad. I got used to use backtick nowadays because of the `${variable}` syntax ✌️